api/simplestore/sessions_test.go

```diff
@@ -0,0 +1,121 @@
+package simplestore
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/18F/e-QIP-prototype/api"
+	"github.com/google/uuid"
+)
+
+func getDateAndUUID() (string, time.Time) {
+	return uuid.New().String(), time.Now().Add(time.Duration(5 * time.Minute))
```

Comment:
The return order (uuid, date) doesn't match the function name (date, uuid)

Reply:
:+1: since `getUUIDAndDate` is hard to read, updating the order & calling code.